src/main/java/com/example/bookstorewebapp/controller/BookController.java

```diff
@@ -16,6 +18,7 @@
 @RequestMapping(value = "/api/books")
 @RequiredArgsConstructor
 public class BookController {
+    private static final String PATH_ID = "/{id}";
```

Comment:
i think it's not a good idea to make endpoint as constant

Reply:
Agree, that actually reduces readability, let's put these values directly as a param of the Mapping